tests/components/comelit/conftest.py

```diff
@@ -47,10 +49,10 @@ def mock_serial_bridge() -> Generator[AsyncMock]:
         ),
     ):
         bridge = mock_comelit_serial_bridge.return_value
-        bridge.get_all_devices.return_value = BRIDGE_DEVICE_QUERY
+        bridge.get_all_devices.return_value = deepcopy(BRIDGE_DEVICE_QUERY)
```

Comment:
What is the reason for `deepcopy` here, it doesn't look like you modify the `BRIDGE_DEVICE_QUERY` constant during the tests

Reply:
This was part of another PR that slipped in...